### ISSUE TITLE
Matrix wallpaper sometimes set to Rainbow colour on startup

### ISSUE THREAD

**Issue by Anonymous-Kingg** [rocksdanister/lively#2483]
**Describe the bug**
Upon booting up my Windows 10 PC, sometimes my matrix wallpaper on my second monitor will have its colour set to Rainbow. If I check the customisation settings, it still shows up as being the colour that I originally set it to. This is sometimes fixed by changing the wallpaper to a different one and then back to matrix, but sometimes requires me to re-select my desired colour in the customisation settings.

**To Reproduce**
1. Select matrix wallpaper and colour
2. Restart computer

**Expected behavior**
For the colour of the wallpaper to be as defined in the customisation settings

**Screenshots/Video**
Happens seemingly at random so screenshots/video is tedious to get.

**Desktop (please complete the following information):**
 - OS: Windows 10 Home
- Matrix wallpaper on second monitor, YouTube video on primary monitor.

**Additional context**
Has been a persistent issue for many months

**Log file (Important)**
[20241121_194543.txt](https://github.com/user-attachments/files/17845576/20241121_194543.txt)

**Comment by CALLxMExACE** [comment]
![Image](https://github.com/user-attachments/assets/c586a359-54b8-4784-838b-d2fcad663c15)
Windows 11. Happens mostly when my monitors wake up on my 2nd monitor. It's annoying have to reset my wallpaper every time I sit down. About ready to uninstall because what's the point?

**Comment by Anonymous-Kingg** [comment]
> ![Image](https://github.com/user-attachments/assets/c586a359-54b8-4784-838b-d2fcad663c15) Windows 11. Happens mostly when my monitors wake up on my 2nd monitor. It's annoying have to reset my wallpaper every time I sit down. About ready to uninstall because what's the point?

You know, I guess I should have mentioned that it happens on my second monitor, but I guess I couldn't have known that it was a problem specifically with that functionality. It actually almost never happens to me anymore, so it's interesting to know that this is still a persistant problem. Are you on the latest version too?

**Comment by CALLxMExACE** [comment]
> > ![Image](https://github.com/user-attachments/assets/c586a359-54b8-4784-838b-d2fcad663c15) Windows 11. Happens mostly when my monitors wake up on my 2nd monitor. It's annoying have to reset my wallpaper every time I sit down. About ready to uninstall because what's the point?
> 
> You know, I guess I should have mentioned that it happens on my second monitor, but I guess I couldn't have known that it was a problem specifically with that functionality. It actually almost never happens to me anymore, so it's interesting to know that this is still a persistant problem. Are you on the latest version too?

Should be. I got it from the microsoft store a few weeks ago and the last update was in 2024 on here. Can't find a way in the app to see what version it's running. The About section doesn't mention it. But yeah I only have it occuring on my 2nd monitor which is a 4k, my primary 2k monitor hasn't done it once.

**Comment by rocksdanister** [comment]
This is an issue with some webpages when getting refreshed Chromium is not able to detect when its finished properly so re-applying the Lively settings fail. Workaround is just to set the default to non rainbow. 
duplicate: https://github.com/rocksdanister/lively/discussions/1749

**Comment by CALLxMExACE** [comment]
> This is an issue with some webpages when getting refreshed Chromium is not able to detect when its finished properly so re-applying the Lively settings fail. Workaround is just to set the default to non rainbow. duplicate: [#1749](https://github.com/rocksdanister/lively/discussions/1749)

If settings the default was the workaround that worked, I wouldn't be here.

I was however able to solve my problem by reading the post. 

Go to your user app data folder, then AppData\Local\Packages\12030rocksdanister.LivelyWallpaper_97hta09mmv6hy\LocalCache\Local\Lively Wallpaper\Library\wallpapers\xn0quq52.bq2

Open the Script.js in notepad. Set the color you want in lines 3-5. Because it loves to not remember. 
Then go to line 8 and change true to false. 

Bam. We fixed what the creator could not manage/was too lazy to fix.

You can also have fun if you are familiar with basic coding. Such as adding additional characters.

**Comment by rocksdanister** [comment]
> If settings the default was the workaround that worked, I wouldn't be here.

No.. this is exactly what I said https://github.com/rocksdanister/lively/discussions/1749#discussioncomment-6109906

> Bam. We fixed what the creator could not manage/was too lazy to fix.

I have explained exactly what is happening here but you seem to not follow or care, not wasting my time further explaining.

Next time search before creating duplicate issue, locking since resolved.
